test/unit/content-src/components/DiscoveryStreamComponents/TopSites.test.jsx

```diff
@@ -23,3 +27,20 @@ describe("Discovery Stream <TopSites>", () => {
     assert.equal(oldTopSites.props().title, "Test");
   });
 });
+
+describe.only("DSCard DS Top Sites", () => {
+  let wrapper;
+  let DEFAULT_PROPS = {
+    header: "test",
+    isFixed: true,
+  };
+
+  beforeEach(() => {
+    const store = createStore(combineReducers(reducers), INITIAL_STATE);
+    wrapper = mountWithIntl(<Provider store={store}><TopSites {...DEFAULT_PROPS} /></Provider>);
```

Comment:
@Mardak Discovery stream component has 75% test coverage calling out connect as uncovered block

Mounting Topsites above is giving error here https://github.com/mozilla/activity-stream/blob/master/content-src/components/TopSites/TopSite.jsx#L418
with this.props.TopSitesRows as undefined. Looks like its set  https://github.com/mozilla/activity-stream/blob/master/content-src/components/TopSites/TopSites.jsx#L172

Is there any way to set state.Prefs.values.topSitesRows from tests to override this error? Thanks

<img width="888" alt="top-site-code-coverage" src="https://user-images.githubusercontent.com/597734/59132688-b59b5b00-892a-11e9-9589-a0fd8cd79e87.png">

Reply:
Just to note out conversation from slack, we need to import the 2 differently exported components to test the connected version vs plain component.

Reply:
@Mardak quick clarification on the feedback, do we want separate unit tests for connected (TopSites) and plain (_TopSites) component. My understanding mounting connected component loads and run tests on plain component, and showing test coverage 100% for TopSites component